8.2.17 Release # DO NOT DELETE

su-sws-jira opened the pull request.

su-sws-jira committed:
8.2.17

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

pookmish committed:
Updated dependencies

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

pookmish committed:
Default configs and content updates (#699)

* updated default configs and content

* updated test

* Adjusted test

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

su-sws-jira committed:
Updated dependencies Aug 26 2020

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

kaseybon committed:
Sparkbox sprint 23 (#691)

* feat(STN-373): added lando and CSS documentation for developers (#671)

* feat(STN-481): update the default color pairing for Colorful to Cardinal (#685)

* fix(STN-492): secondary nav header spacing and border-top (#684)

* fix(STN-492): secondary nav header spacing and border-top

* fix(STN-492): slight margin adjustment to Colorful

* chore(STN-492): compiled CSS

* STN-480:  Fix spacing beside navigation dropdown toggle (#686)

* fix (STN-480): add spacing to navigation dropdown arrow in nested toggler

- When all pages within a nested toggler are private, the class of hb-main-nav__item--parent is not added. Without this class the icon beside nav items doesn't recieve appropriate spacing. Resolve by adding spacing to the nested toggler.

* feat: add compiled CSS

* STN-495: Add the Warbler Color Pairing (#688)

* feat(STN-495): warbler color pairing and local dark footer changes for tradl

* feat(STN-495): warbler theme settings

* chore(STN-495): compiled css

* feat (STN-490): refactor hb-grid system

JIRA STN-490 https://sparkbox.atlassian.net/browse/STN-490

* feat: refactor paragraph type grid system to use CSS Grid

* Feat 392  behat (#680)

* wip: starting on behat test for row style

* update: row paragraph test add spotlight test

Co-authored-by: Lindsay Silver <lindsay@heysparkbox.com>

* feat (STN-493): add title region to 3 column w/image layout

JIRA STN-493 https://sparkbox.atlassian.net/browse/STN-493

* feat (STN-522): create utility class to display title text on Colorful arrow link

JIRA STN-522 https://sparkbox.atlassian.net/browse/STN-522

* STN-520: spacing bug fix for hb-views-divider class (#697)

* fix(STN-520): adjust views divider hb-card spacing from qa feedback

I added padding to the hb-views-class, and removed the margin.  Padding will apply within the container above the divider being added and not outside of the container as margin does.

* chore: compiled css

* chore: compiled CSS

* fix: Include the new title region in the 3 col with image layout module

Co-authored-by: Heather Taylor <heather@heysparkbox.com>
Co-authored-by: Lindsay R. Silver <lindsay@heysparkbox.com>
Co-authored-by: KelbelR <kelbel@heysparkbox.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

pookmish committed:
Added cancel button to entity forms

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

pookmish committed:
HSD8-869 Adjusted the title and department to import separately

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

pookmish committed:
HSD8-889 Slightly improve media library form

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

pookmish committed:
HSD8-886 Disable the preview button on all node types

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

pookmish committed:
Fixed tests

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

pookmish committed:
HSD8-850 Improved events importer form. (#706)

* HSD8-850 Use a better events importer UI

* cc fixes

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr693-jvxfzyme9skzvoxwg08ephm1vev9ujtq.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143

Visual Diffs:
- https://dashboard.tugboat.qa/5f3d5a33c121de7f0710d143/visualdiffs

pookmish committed:
Dont require event type in the form
